.editorconfig

```diff
@@ -20,4 +20,4 @@ indent_size = 2
 trim_trailing_whitespace = false
 
 [*.yml]
-indent_size = 4
+indent_size = 2
```

Comment:
This rule is not necessary then, there are 2 spaces by default. Therefore the entire [*.yml] and also [package.json] sections can be removed.

Reply:
Ok, removed.